jobs/JEVS_GLOBAL_ENS_STATS

```diff
@@ -1,82 +1,145 @@
+export VERIF_CASE=${VERIF_CASE:-grid2obs}
+export MODELNAME=${MODELNAME:-gefs}
+export modsys=${modsys:-gefs}
+export mod_ver=${mod_ver:-${gefs_ver}}
 
-export SENDCOM=${SENDCOM:-YES}
-export SENDDBN=${SENDDBN:-YES}
-export SENDECF=${SENDECF:-YES}
-export SENDMAIL=${SENDMAIL:-NO}
-
-export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}
-export EVSIN=${EVSIN:-$COMIN/prep/$COMPONENT/$RUN}
-export EVSINwmo=${EVSINwmo:-$COMIN/prep/$COMPONENT/atmos}
-export COMOUT=${COMOUT:-$(compath.py -o $NET/$evs_ver)}/$STEP/$COMPONENT
-mkdir -p $COMOUT
-
-export COMOUTsmall=$COMOUT/$RUN.$VDATE/$MODELNAME/$VERIF_CASE
-export COMOUTfinal=$COMOUT/$MODELNAME.$VDATE
-
-mkdir -p $COMOUTsmall $COMOUTfinal
+# Run setpdy and initialize PDY variables
+if [ "${RUN}" == "chem" ]; then
+    setpdy.sh 8
+    . ./PDY
+else
+    setpdy.sh 
+    . $DATA/PDY 
+fi
 
 export HOMEevs=${HOMEevs:-${PACKAGEROOT}/${NET}.${evs_ver}}
-export EXECevs=$HOMEevs/exec
-export PARMevs=$HOMEevs/parm
-export USHevs=$HOMEevs/ush
-export SCRIPTerf=$HOMEevs/scripts
-export FIXevs=${FIXevs:-$HOMEevs/fix}
-
+export EXECevs=${EXECevs:-${HOMEevs}/exec}
+export PARMevs=${PARMevs:-${HOMEevs}/parm}
+export FIXevs=${FIXevs:-${HOMEevs}/fix}
+
+if [ "${RUN}" == "chem" ]; then
+    export SCRIPTSevs=${SCRIPTSevs:-${HOMEevs}/scripts}
+    export USHevs=${USHevs:-${HOMEevs}/ush/${COMPONENT}}
+    export MASK_DIR=${MASK_DIR:-${FIXevs}/masks}
+else
+    export USHevs=$HOMEevs/ush
+    export SCRIPTerf=$HOMEevs/scripts
+fi
 
-####################################
-# Determine Job Output Name on System
-####################################
-export outid="LL$job"
-export jobid="${outid}.o${pid}"
+## May need a default setting to avoid non-defined VDATE
+if [ "${RUN}" == "atmos" ] || [ "${RUN}" == "wmo" ] || [ "${RUN}" == "headline" ] ; then
+    export VDATE=${VDATE:-$PDYm2}
+elif [ "${RUN}" == "chem" ]; then
+    export VDATE=${VDATE:-${PDYm3}}
+fi
 
+export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
+if [ "${RUN}" == "chem" ]; then
+    export EVSINgefs=${EVSINgefs:-${COMIN}/prep/${COMPONENT}}
+    export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})}
+    export COMOUTsmall=${COMOUTsmall:-${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}/${VERIF_CASE}}
+    export COMOUTfinal=${COMOUTfinal:-${COMOUT}/${STEP}/${COMPONENT}/${MODELNAME}.${VDATE}}
+    
+    mkdir -p ${COMOUT} ${COMOUTsmall} ${COMOUTfinal}
+    
+    export DATA_TYPE=${DATA_TYPE:-aeronet}
+     
+    # alternative choice is an option different from aqobs
+    export airnow_hourly_type=${airnow_hourly_type:-aqobs}
+else
+    export EVSIN=${EVSIN:-$COMIN/prep/$COMPONENT/$RUN}
+    export EVSINwmo=${EVSINwmo:-$COMIN/prep/$COMPONENT/atmos}
+    export COMOUT=${COMOUT:-$(compath.py -o $NET/$evs_ver)}/$STEP/$COMPONENT
+    mkdir -p $COMOUT
+    
+    export COMOUTsmall=$COMOUT/$RUN.$VDATE/$MODELNAME/$VERIF_CASE
+    export COMOUTfinal=$COMOUT/$MODELNAME.$VDATE
+    
+    mkdir -p $COMOUTsmall $COMOUTfinal
```

Comment:
Move `mkdir -p ${COMOUT} ${COMOUTsmall} ${COMOUTfinal}` outside the if/else/fi and remove `mkdir -p $COMOUT` and `mkdir -p $COMOUTsmall $COMOUTfinal` from the else section.